audio/build.zig

```diff
@@ -22,19 +14,19 @@ pub fn build(b: *Builder) void {
     const soundio_tests = b.addTest("soundio/main.zig");
     soundio_tests.setBuildMode(mode);
     soundio_tests.addPackage(pkg);
-    link(b, soundio_tests, .{ .alsa = true });
+    link(b, soundio_tests);
 
     const test_step = b.step("test", "Run library tests");
     test_step.dependOn(&soundio_tests.step);
 }
 
-pub fn link(b: *Builder, step: *std.build.LibExeObjStep, options: SoundIoOptions) void {
-    const soundio_lib = buildSoundIo(b, step, options);
+pub fn link(b: *Builder, step: *std.build.LibExeObjStep) void {
```

Comment:
I do think we should still take an options struct in this function parameter, just so we can add any options necessary in the future without breaking user's current use of `audio.link(...)`. It should just be an empty `Options` struct for now, though.